dinky-common/src/main/java/org/dinky/utils/SqlUtil.java

```diff
@@ -45,7 +45,7 @@ public static String[] getStatements(String sql, String sqlSeparator) {
             return new String[0];
         }
 
-        String[] splits = sql.replace(";\r\n", ";\n").split(sqlSeparator);
+        String[] splits = sql.replace(";\r\n", ";\n").split(";");
```

Comment:
should use common sql split character at here? this seems influence 'if (lastStatement.endsWith(SEMICOLON))'

Reply:
Sorry, here is another bug, should not be in this pr, I accidentally put the temporary change up, here I will revert